### ISSUE TITLE
prevents user logins

### ISSUE THREAD

**Issue by bremser** [DLXPlugins/dlx-pmpro-turnstile#4]
Easy set up and working well with checkout form, but when it's on for checkout, it interferes with user login. Every login attempt says password incorrect. Have WP Login, PMP Login, PMP Password reset, WP reset all set to off. Removed PMP Askimet plugin and issue remained.

**Comment by ronalfy** [comment]
@bremser do you have two-factor enabled?

I recommend using the default login form if you're logging in as admin. It would be `https://yourdomain.com/wp-login.php`

**Comment by bremser** [comment]
I have 2FA (via Wordfence), but it is only set for admins. The issue was for all subscriber logins using the PMP login page. The error message is incorrect password, nothing about 2FA or Turnstile. The only other login related plugin I have is "WordPress Persistent Login", which keeps users logged in for longer than 30 days. 

When I did turn on Turnstile for PMP login page to see if that would resolve it and it did not:
<img width="796" alt="Screen Shot 2023-12-13 at 12 22 36 PM" src="https://github.com/DLXPlugins/dlx-pmpro-turnstile/assets/3586749/669ef33e-f657-4601-88cc-6dde22f6dd5c">

(I did test Turnstile with checkout and it worked as expected)

**Comment by ronalfy** [comment]
Thanks. I need to test this on my live site. I do know that the 2FA on the
WP login screen works with WordFence.

**Comment by ronalfy** [comment]
@bremser I've released 1.0.5. I was able to get this working on one of my live sites. Can you please give it a try?

You can download the zip here: https://github.com/DLXPlugins/dlx-pmpro-turnstile/releases/tag/1.0.5

**Comment by bremser** [comment]
Same behavior - I realized when installing I'm not on the latest version PMP so I'll update that and then test again.

**Comment by ronalfy** [comment]
@bremser did you try logging in as a member? I know for admins, the PMPro login page won't work with WordFence. You have to go to `https://yourdomain.com/wp-login.php` to log in as an admin.

**Comment by bremser** [comment]
Yes, I have a test subscriber role that I am testing with the PMP login page. Didn't test admin, but will test that at the same time with the wp-login url to see how that behaves.

**Comment by ronalfy** [comment]
I noticed with my site it could be caching.
